cogs/battling.py

```diff
@@ -439,7 +439,7 @@ async def on_move_request(self, cluster_idx, user_id, species_id, actions):
         embed = self.bot.Embed(title=f"What should {species} do?")
 
         embed.description = "\n".join(
-            f"{k} **{v['text']}** • `p!battle move {v['command']}`" for k, v in actions.items()
+            f"{k} **{v['text']}** • `{ctx.prefix}!battle move {v['command']}`" for k, v in actions.items()
```

Comment:
```suggestion
            f"{k} **{v['text']}** • `p!battle move {v['command']}`" for k, v in actions.items()
```

Reply:
There is no `ctx` here and this is in DMs.

Reply:
Oh alright, I'll fix the other too. My bad